### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by serge2016** [Ensembl/Bio-DB-HTS#106]
Hello!

Don't you mind to make a new tag? And also what do you think about releases instead of tags?

**Comment by tillea** [comment]
I fully agree.  I just updated the Debian package which formerly was watching [CPAN](https://metacpan.org/release/Bio-DB-HTS) to the latest tag which is here 3.02 while CPAN has 3.01.  It would be great if releases would be consistent and transparent when new code is of release quality.
